test/cypress/integration/07-signin.test.js

```diff
@@ -8,7 +8,6 @@ describe('signin', () => {
     cy.visit('/signin?next=/testuseradmin');
     cy.get('input[name=email]').type('testuser+admin@opencollective.com');
     cy.get('button[type=submit]').click();
-    cy.getByDataCy('topbar-login-username').contains('Test User Admin', { timeout: 15000 });
```

Comment:
Can we restore that since you've moved back the icon?

Reply:
@Betree : Ops, sorry, I've missed this. I've added it back. 🤔